docs/NetworkAPI_Links.md

```diff
@@ -0,0 +1,169 @@
WVj8oahrdhc+XnOzfGzbfwziuk1jxZrfi3xt/wAIt8PL0WlvpsoOt62sSSrAf+feMOCrSHvwdv1BFefePLc3nwH+FlsJpYDNeadGJYW2vHmBhuU9iM5BrdsID8A/GEFk88sngLXGWNJp33HTrvAGXbptcAknj8Nh3AHtUSNHCiPI0rKoBkcDc59TgAZPsAK5LXfCvh2z8Vv8QvENxcSNpNgyok2JILRF+cyogUtv68gnr0zjHYVwut/FCz8L/EaDw54otBpenXlt5tlrU0+IZnBAaNsqAhHOSW4+XONwoA4HW/EOh/Fz4peEU+H7pc3OiXK399qrL5Bjt1fmIK+15Mkc4BUbx/ebHvFeFfFrX9B8UeIvCFr4D1Cz1PxauqxNDc6bIsrQ24Dlw8iZG3OGKk9Ax6Zz7rQAUUUUAFFFFABRRRQAUUUUAFFFFABRRRQAUUUUAFFFFABRRRQAUUUUAFFFFABRRRQAUUUUAFFFFABRRRQAUUUUAFFFFABRRRQAUUUUAFFFFABRRRQAUUUUAFFFFABRRRQAUUUUAFFFFABRRRQAUUUUAFZOreFPD2vXKXGuaDpmpTImxJbyzjmZVyTgFgSBkk4961qKAOb/AOFceCP+hN8P/wDgrg/+JrX/ALG0v+2P7W/s20/tLy/K+2+QvnbP7u/G7HtmrtFAFOx0fTNMuLqfTdOtLOa8kMtzJbwLG075J3OQPmOSeTzyaRdG0xdYbVl020GpNH5TXogXzin90vjdjgcZ7VdooApafo2maTJcPpWm2lk91J5tw1tAsZmf+820DceTyabb6DpFpq8+q2mlWMGo3ClZryK2RZpRwcM4G4j5R1PYelX6KAI7i3hu7WW2u4Y54JkMcsUqhldSMFSDwQRxg1z/APwrjwR/0Jvh/wD8FcH/AMTXSUUAZ+k+H9G0COSPQtJsdMSUhpFs7ZIQ5HQkKBmtCiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKwbzwL4R1G8lu9Q8LaLdXMzbpJp9Oid3PqWK5JreooAxNP8FeFdIv477SvDWj2N3Fny7i2sIo5EyCDhlUEZBI+hq3DoGj21re2tvpNjFb37O13ElsipclxhjIAMOSODnOa0KKAILOytdOs4rTT7aG1toV2xwwRhEQegUcAVWh8P6Nb2V3Z2+kWMVrfMz3UCWyKlwzDDF1AwxI4Oc5rQooAgtrG0srGOys7WG3tI08tIIowsaL/AHQo4A9qh0rRtL0K0Nromm2mnW7OXMNpAsSFiAC21QBnAHPtV2igDP1bw/o2vxxx67pNjqaREtGt5bJMEJ6kBgcVl/8ACuPBH/Qm+H//AAVwf/E10lFAEFnZWunWcVpp9tDa20K7Y4YIwiIPQKOAKnoooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACmSxRzwvDPGskUilXR1yrA8EEHqKfRQBzf/CuPBH/AEJvh/8A8FcH/wATVyy8I+G9Os7u10/w9pVrbXqhLqGCyjRJ1GcB1C4YfMeDnqfWtiigDOm8PaLcafa2FxpFhLZ2bq9tbvbI0cDKCFKKRhSASAR0zVu7s7bULOW0v7eK6tplKSwzIHR1PUFTwR9amooAz5vD+jXFlaWdxpFjLa2LK9rA9sjJbsowpRSMKQOBjGKm1LS7DWLF7LV7G2v7VyC0F1CsqMQcjKsCDg81aooAjt7eG0tore1iSGCFAkcUahVRQMBQBwABxiodS0uw1ixey1extr+1cgtBdQrKjEHIyrAg4PNWqKAMzSPDWhaA0raDounaY0wAlNlaJCXA6Z2gZxk9fWtOiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigD/9k=
```

Comment:
? 
I don't see any image here (but it may be due to some adblock or so..)

Reply:
Hmmm, you are right, the image does not show when viewed with a browser on GitHub.  It looks fine in my .md editor.
Perhaps the image needs to be a separate file.  I will try to find a better solution.